src/tigerbeetle.zig

```diff
@@ -248,6 +248,26 @@ pub const CreateTransfersResult = extern struct {
     }
 };
 
+pub const LookupAccountResult = enum(u32) {
```

Comment:
Since the only place we use these will be in the REPL, they might go better there, but I'm not sure just yet.

Reply:
I'd be happy to move them over, just let me know!

Reply:
@ReppCodes thanks, can you please move them over to `repl.zig`. We chatted a bit internally, since how we handle account / transfer lookups is likely to change once queries land, so it'll be good to have them there instead!

Reply:
Done!